src/ui/models/windowstate.coffee

```diff
@@ -0,0 +1,19 @@
+Client = require 'hangupsjs'
+settings = require('node-persist')
```

Comment:
Why would it be better to use node-persist instead of `localStorage`

Reply:
localStorage requires an active BrowserWindow, but since this code is used to initialise the position of the BrowserWindow there's no way of actually using that. That's why node-persist is required.